example/tests/test_models.py

```diff
@@ -274,3 +274,6 @@ def test_notification_should_be_filtered_with_related_object(self):
         self.assertEqual(Notification.objects.filter_with_related_object(self.author)[0], self.notification)
         # recipient is not related object
         self.assertFalse(Notification.objects.filter_with_related_object(self.recipient).exists())
```

Comment:
I left the `pynotify.models.Notification` in tests.

Now I am thinking about whether a custom `Notification` model should not deserve its own test case. 🤔